brian2/devices/cpp_standalone/device.py

```diff
@@ -954,13 +960,24 @@ def compile_source(self, directory, compiler, debug, clean):
                 with std_silent(debug):
                     if vcvars_cmd:
                         if clean:
+                            start_time = time.time()
                             os.system(f'{vcvars_cmd} >>winmake.log 2>&1 && {make_cmd} clean > NUL 2>&1')
+                            self.timers['compile']['clean'] = time.time() - start_time
+                        start_time = time.time()
                         x = os.system(f'{vcvars_cmd} >>winmake.log 2>&1 && {make_cmd} {make_args}>>winmake.log 2>&1')
+                        self.timers['compile']['make'] = time.time() - start_time
                     else:
                         os.environ.update(msvc_env)
                         if clean:
+                            start_time = time.time()
                             os.system(f'{make_cmd} clean > NUL 2>&1')
+                            self.timers['compile']['clean'] = time.time() - start_time
+                        start_time = time.time()
                         x = os.system(f'{make_cmd} {make_args}>>winmake.log 2>&1')
+                        self.timers['compile']['make'] = time.time() - start_time
+                    self.timers['compile']['all'] = sum(
+                        t for t in self.timers['compile'].values() if t is not None
+                    )
```

Comment:
This seems a bit unnecessarily complex to sum up the two values. I don't think we need to store "all" in the dictionary, since it is easy to calculate by summing up "clean" and "make", and in most cases "make" is what we are interested in mostly.

Reply:
Sure, just did this for the benchmarks (had to happen somewhere). I'll make changes next week!

Reply:
> I'll make changes next week!

:+1: Nothing very urgent, obviously.